public/pages/Alerts/containers/Alerts/Alerts.tsx

```diff
@@ -76,15 +81,21 @@ export interface AlertsState {
   groupBy: string;
   recentlyUsedRanges: DurationRange[];
   selectedItems: AlertItem[];
+  correlatedItems: CorrelationAlertTableItem[];
   alerts: AlertItem[];
+  correlationAlerts: CorrelationAlertTableItem[];
   flyoutData?: { alertItem: AlertItem };
+  flyoutCorrelationData?: { alertItem: CorrelationAlertTableItem };
   alertsFiltered: boolean;
+  filteredCorrelationAlerts: CorrelationAlertTableItem[];
```

Comment:
We should club all these fields and instead have a single field called `tabState` or something which gets updated on tab switch. That way the UI will automatically refresh when the state changes and we don't need to duplicate so much code below

Reply:
will raise a seperate PR to refactor this.